.env.babylon-integration.example

```diff
@@ -11,16 +11,11 @@
 
 ### Bitcoin Configuration
 BITCOIN_NETWORK=signet
-BITCOIN_RPC_HOST=11.22.33.44:38332 # gRPC
-BITCOIN_RPC_USER=snapchain
+BITCOIN_RPC_IP=11.22.33.44
+BITCOIN_RPC_PORT=38332
```

Comment:
i feel it's better to have two vars:

```
BITCOIN_NODE_IP=11.22.33.44 # used by btc-staker (devnet-only)
BITCOIN_RPC_HOST=11.22.33.44:38332 # gRPC. used by FG
```

b/c once we deploy to Bitcoin mainnet, we likely don't want to maintain a mainnet Bitcoin node for our FG

Reply:
yes, i thought of this before, but forgot it when i last edited it 😂